dataengineeringutils/glue.py

```diff
@@ -182,27 +182,31 @@ def get_glue_column_spec_from_metadata(metadata):
     return glue_columns
 
 
-def metadata_to_glue_table_definition(metadata):
```

Comment:
We now need both the table and the database metadata to create a table in the glue catalogue, because we need to combine the absolute path in the db with the relative path in the table